builder/qemu/step_type_boot_command.go

```diff
@@ -157,11 +167,7 @@ func vncSendString(c *vnc.ClientConn, original string) {
 			log.Printf("Special code '<leftAltOn>' found, replacing with: %d", keyCode)
 
 			c.KeyEvent(keyCode, true)
-			time.Sleep(time.Second / 10)
```

Comment:
Yeah I think the 100ms was a bug / bad copy-paste job since it was coded in a really odd way. I'll see if I can figure out when that was introduced.